src/styles/landing.module.scss

```diff
@@ -0,0 +1,116 @@
+@mixin centerStuff {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+@mixin border_outline() {
+  border: none;
+  outline: none;
+}
+@import "./dropdown";
+
+$netflixColor: #e50914;
+$netflixGroundColor: #0d0c0d;
+
+* {
+  font-family: "Netflix Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+p {
+  color: rebeccapurple;
+  font-size: larger;
+}
+
+.topContainer {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.651), rgba(0, 0, 0, 0.562)),
+    url("../images/cover.jpg");
+  height: 95vh;
+  width: 100%;
+
+  .navbar {
+    @include centerStuff();
+    justify-content: space-between;
+    padding: 2rem 5rem;
+
+    .logo {
+      width: 12rem;
+      height: 3.5rem;
+    }
+    .buttons {
+      @include centerStuff();
+
+      .dropdown {
+        padding: 0 30px;
+        .list {
```

Comment:
very dangerous! hard to fix it `topContainer > navbar > button > dropdown > list` 5 layer here max should be 3 more than 3 is dangerous